tests/functional/modules/test_zos_backup_restore.py

```diff
@@ -574,6 +596,8 @@ def test_backup_and_restore_exclude_from_pattern(ansible_zos_module):
             backup_name=data_set_backup_location,
         )
         assert_module_did_not_fail(results)
+        for result in results.contacted.values():
+            assert result.get("backup_name") == data_set_backup_location  # Critical Line
```

Comment:
Some assertions look inconsistent, sometimes there is use of the assertion message like this

```
assert result.get("backup_name") == backup_name, \
                f"Backup name '{backup_name}' not found in output"
```

And sometimes there is not, please adhere to one. Also, what does Critical Line means here?